harness/propertyHelper.js

```diff
@@ -140,7 +140,8 @@ function isSameValue(a, b) {
 }
 
 function isWritable(obj, name, verifyProp, value) {
-  var newValue = value || "unlikelyValue";
+  var unlikelyValue = Array.isArray(obj) && name === "length" ? 217493 : "unlikelyValue";
```

Comment:
rather than modifying this helper, why not pass in the value from the test?

Reply:
The test uses convenient `verifyProperty` helper, which doesn't allow passing a `value`.

Reply:
Perhaps that would be a better helper change than this one?

Reply:
As a test author, I have an expectation that both `verifyWritable` and `verifyProperty` would be as less intrusive/observable as possible and would not throw any non-related errors.

It is a bit hard to achieve with Array length, as setting `length` may delete elements, so this change now uses `2 ** 32 - 1` for _unlikely_ length.